installation/production_run.md

```diff
@@ -218,21 +238,36 @@ Reference the following files for detailed configuration:
    - Verify DNS resolution works
    - Test internal container networking
 
-4. **Firebase Configuration**
+4. **Invalid Google Cloud Service Account**
+
+Check Marble's startup logs for messages related to Google Cloud Platform's authentication, they may be able to point your to a potential misconfiguration:
+
+For example, a good configuration would look like this:
+
+```
+2025-05-27T14:12:41+02:00 INFO successfully authenticated in GCP principal=marble-dev@my-projectiam.gserviceaccount.com project=my-project
+2025-05-27T14:12:41+02:00 INFO FIREBASE_PROJECT_ID was not provided, falling back to Google Cloud project project=my-project
+2025-05-27T14:12:41+02:00 INFO firebase project configured project=my-project
+```
+
+Those lines indicate:
+
+ - The authenticated service account is `marble-dev@my-project@iam.gserviceaccount.com project=my-project`
+ - The detected Google Cloud Project is `my-project`
+ - The assumed Firebase project is also `my-project`
+
+You can verify that those value match your environment if you encounter any issue.
+
+5. **Firebase Configuration**
 
-   - Service Account Key:
+  - Service account:
+    - Check that the detected Google Cloud project and service account match your environment
 
-     - ⚠️ Backend service will fail to start without accessible Firebase service account key
-     - Exception: On GCP, automatic service account discovery may work
-     - Generate key in Firebase Console → Project Settings → Service Accounts
-     - Mount key file to container and set `GOOGLE_APPLICATION_CREDENTIALS`
-     - Example: `GOOGLE_APPLICATION_CREDENTIALS=/secrets/firebase-key.json`
+  - Required environment variables:
+    - `FIREBASE_API_KEY`: Web API key from Firebase Console
 
-   - Required Environment Variables:
-     - `GOOGLE_CLOUD_PROJECT`: Firebase project ID
-     - `FIREBASE_API_KEY`: Web API key from Firebase Console
-     - `FIREBASE_AUTH_DOMAIN`: Auth domain from Firebase settings
-     - `FIREBASE_APP_ID`: Application ID from Firebase Console
+  - If you plan on using Single-Sign On (SSO) with Firebase (to delegate authentication to another Identity Provider), you will need to configure the following directives:
+    - `FIREBASE_AUTH_DOMAIN`: Auth domain from Firebase settings
```

Comment:
Removes app ID, which is not required at all, and moved the auth domain as optional, since it's only used for SSO.